### PR TITLE
__libvirtd: new type

### PR DESCRIPTION
**Pull request by sideeffect42**
Adds a new type `__libvirtd` which installs the libvirt system daemon and packages for running KVM VMs on the native architecture.

This type is only tested on Debian (and some testing on Arch Linux has been done some time ago).

### CODE REVIEW

File: type/__libvirtd/man.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+=======================
+
+NAME
+----
+cdist-type__libvirtd - Manage the libvirt hypervisor
+
+
+DESCRIPTION
+-----------
+This type can be used to mananage the `libvirt <https://libvirt.org>`_ hypervisor.
```
Comment (4nd3r) [inline type/__libvirtd/man.rst:11]: typo

File: type/__libvirtd/manifest [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+		# NOTE: ebtables is required for some network filtering options
+		__package ebtables
+
+		# __update_alternatives iptables --path /usr/sbin/iptables-legacy
+
+		__package qemu-utils
+
+		# NOTE: We have to install netcat-openbsd because libvirt's developers
+		# are are complete idiots ignoring the fact that there are different
```
Comment (4nd3r) [inline type/__libvirtd/manifest:59]: let's not say bad things about people

File: type/__libvirtd/man.rst (hunk 1 above)
Comment (sideeffect42) [reply type/__libvirtd/man.rst]: thanks.

File: type/__libvirtd/manifest (hunk 2 above)
Comment (sideeffect42) [reply type/__libvirtd/manifest]: Whoops, looks like I didn't mean to publish this type when I wrote it 😄.

**Review by 4nd3r — APPROVED** [review]

**Pull request merged**